edgraph/server.go

```diff
@@ -1102,6 +1102,16 @@ func getAuthMode(ctx context.Context) AuthMode {
 // QueryGraphQL handles only GraphQL queries, neither mutations nor DQL.
```

Comment:
I am not sure if this comments is still valid. Please check and suggest should I fix this

Reply:
This comment is still valid.